.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master, develop ]
```

Comment:
We don't use the `master` branch, the `main` instead of it.

Reply:
Done